docs/sources/clients/promtail/_index.md

```diff
@@ -39,10 +39,34 @@ drop, and the final metadata to attach to the log line. Refer to the docs for
 
 ### Support for compressed files
 
-Promtail now has native support for ingesting compressed files by a mechanism that
-relies on file extensions. If a discovered file has an expected compression file
-extension, Promtail will **lazily** decompress the compressed file and push the
-parsed data to Loki. Important details are:
+Promtail now has native support for ingesting compressed files by a mechanism
+that relies on file extensions. If a discovered target has an expected
+compression file extension and decompression is enabled, Promtail will
+**lazily** decompress the compressed file and push the parsed data to Loki.
+Example of YAML configuration to decompress files:
```

Comment:
Is this a Loki configuration or a Promtail configuration file?  There are multiple yaml files floating around, and it's helpful to be specific when we're talking about editing them.

Reply:
Thank you, I pushed a commit addressing it. After your comment I just noticed I missed to add the new configuration to Promtail config docs, let me do it too :smile: